### PR TITLE
Task/api call from frontend

### PR DESCRIPTION
**Pull request by nataliemona**
To make the api call, run `ctgov.get(endpoint)` or `ctgov.post(endpoint)`

Example:
```
  const [exampleInfo, setExampleInfo] = useState([]);
  const fetchExample = async () => {
    try {
      const response = await ctgov.get('brief_summaries');
      setExampleInfo(response.data);
    } catch (err) {
      console.log(err);
    }
  };

  useEffect(() => {
    fetchExample();
  }, []);
```
  
  Then access results with `exampleInfo`

### CODE REVIEW

File: backend/core/settings.py [hunk 1]
```diff
@@ -56,6 +58,11 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+CORS_ALLOW_CREDENTIALS = True
+
+# Whitelist the dev frontend
+CORS_ORIGIN_WHITELIST = ["http://localhost:3000"]
```
Comment (nataliemona) [inline backend/core/settings.py:70]: CORS stuff required to get API call to work in dev localhost environment

File: docker-compose.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ services:
     working_dir: /usr/src/app
     volumes:
       - ./backend:/usr/src/app
-    command: /bin/bash -c "pip-compile && pip install -r requirements.txt && python manage.py migrate && pip list && python manage.py runserver 0.0.0.0:8000"
+    command: /bin/bash -c "pip-compile --quiet && pip install -r requirements.txt -q && python manage.py migrate && pip list && python manage.py runserver 0.0.0.0:8000"
```
Comment (nataliemona) [inline docker-compose.yml:19]: Just making docker output a bit less noisy...

File: frontend/package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
     "antd": "^4.13.1",
+    "axios": "^0.21.1",
```
Comment (nataliemona) [inline frontend/package.json:11]: Library for making API calls

File: frontend/src/App.js [hunk 4]
```diff
@@ -1,30 +1,18 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// Import local files
```
Comment (nataliemona) [inline frontend/src/App.js:5]: Removed the App.js boilerplate code and added in a top level App component which returns the `TopLevelSearchView`. (But later will return different components based on state.) App Component is the main component in React which acts as a container for all other components.

File: frontend/src/index.js [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
 import './index.css';
-// import App from './App';
-import TopLevelSearchView from './views/TopLevelSearchView/TopLevelSearchView';
+import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <TopLevelSearchView />
+    <App />
```
Comment (nataliemona) [inline frontend/src/index.js:12]: Render App component.

File: frontend/src/views/TopLevelSearchView/TopLevelSearchView.spec.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (nataliemona) [inline frontend/src/views/TopLevelSearchView/TopLevelSearchView.spec.js:1]: Really basic unit test to demo how to write unit tests.
We should write more :)

**Review by elarens — APPROVED** [review]

**Review by bapower — APPROVED** [review]

**Review by bapower — COMMENTED** [review]
Everything looks good code-wise.
When I view the front end this is what I see:
Screen Shot 2021-03-16 at 22 16 06
I have run npm install and axios is in my node_modules directory. Is there anything else I need to do to get this to work?

**Pull request merged**